docs/index.asciidoc

```diff
@@ -129,7 +129,13 @@ number of times to retry a failed resolve/reverse
   * Value type is <<array,array>>
   * There is no default value for this setting.
 
-Use custom nameserver(s). For example: `["8.8.8.8", "8.8.4.4"]`
+Use custom nameserver(s). For example: `["8.8.8.8", "8.8.4.4"]`.
```

Comment:
Should we mention somewhere here that specifying a custom DNS server will only resolve fqdns ?

Reply:
pushed clarification LMKWYT